api/rpc/stat/stat.go

```diff
@@ -29,9 +29,16 @@ func New(cfg Config) (*Stat, error) {
 	return &Stat{conn: c}, err
 }
 
-// ExecuteQuery implements business logic for StateServer interface
+// ExecuteQuery implements business logic for StatServer interface
```

Comment:
I would absolutely not expect implementation of an rpc interface to be in `data/influxdb`. This isn't the right abstraction. That belongs in `stat.go`, which should be a consumer of this library package.